tests/test_data_utils.py

```diff
@@ -765,6 +770,66 @@ def test_with_truncation(self):
         }
         dataset = pack_dataset(dataset, seq_length, strategy="bfd")
         self.assertEqual(dataset.to_dict(), expected_output)
+        
+    def test_bfd_packing_power_of_2(self):
```

Comment:
The test cases above this one do test `bfd` for powers of 2, so I think we can remove this specific test case.